Frameworks/include/CoreTextInternal.h

```diff
@@ -32,38 +36,48 @@
 
 @interface _CTFrameSetter : NSObject {
 @public
-    idretaintype(_CTTypesetter) _typesetter;
+    StrongId<_CTTypesetter> _typesetter;
 }
 @end
 
 @interface _CTFrame : NSObject {
 @public
     _CTFrameSetter* _frameSetter;
     CGRect _frameRect;
+    StrongId<NSMutableArray> _lines;
+
+    // TODO::
+    // Do we need these anymore?
     CGSize _totalSize;
-    idretaintype(NSMutableArray) _lines;
     std::vector<CGPoint> _lineOrigins;
 }
 @end
 
 @interface _CTLine : NSObject {
 @public
-    NSRange _strRange;
+    _CTTypesetter* _typeSetter;
```

Comment:
Not any mote. I have these changed to DWrite objects in the next CR. I will clean these up as part of that change

---
In reply to: [77732812](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77732812) [](ancestors = 77732812)